src/main/java/cc/cosmetica/cosmetica/cosmetics/model/Models.java

```diff
@@ -105,7 +105,7 @@ public UnbakedModel getModel(ResourceLocation resourceLocation) {
 					@Override
 					@Nullable
 					public BakedModel bake(ResourceLocation resourceLocation, ModelState modelState) {
-						return this.getModel(resourceLocation).bake(this, l -> sprite, modelState, resourceLocation /*this resource location in bake is just used for debugging in the case of errors*/);
```

Comment:
why remove the comment lol

Reply:
the resourceLocation the comment was referring to was removed no?

Reply:
oh you are right lol